.github/workflows/tests.yml

```
@@ -156,3 +156,27 @@ jobs:
         name: Run stress
         run: |
           docker run chainsafe/gossamer:test sh -c "make it-grandpa"
```

I know this wasn't part of this PR's changes, but I see that there are two tests with name `Run stress`, perhaps one should be named `Run grandpa`?

```suggestion
        name: Run grandpa
        run: |
          docker run chainsafe/gossamer:test sh -c "make it-grandpa"
```